### DISCUSSION TITLE
onErrror event doesn't fire on NextImage in 12.1.5

### DISCUSSION THREAD

**Discussion by danyabunkov** [vercel/next.js#36574] [Help]
### Verify canary release

- [ ] I verified that the issue exists in Next.js canary release

### Provide environment information

  "dependencies": {
    "next": "12.1.5",
    "react": "18.1.0",
    "react-dom": "18.1.0"
  },
  "devDependencies": {
    "@types/node": "17.0.30",
    "@types/react": "18.0.8",
    "@types/react-dom": "18.0.3",
    "eslint": "8.14.0",
    "eslint-config-next": "12.1.5",
    "typescript": "4.6.4"
  }
  node  v16.13.2
  macOs BigSur version 11.3.1

### What browser are you using? (if relevant)

Google Chrome, v 96.0.4664.93

### How are you deploying your application? (if relevant)

-

### Describe the Bug

i just use CLI and get started develop.
i use "yarn create next-app --typescript"
I then use the image from next/image and use the onError event to show the thumbnail if the image fails to load, but the event doesn't fire. Then I tried using the event on the native img and it didn't work either.

P.S. Btw, sorry for my Eng)

### Expected Behavior

onError should work and i should see log.

### To Reproduce

Look at this example
https://codesandbox.io/s/next-image-with-state-forked-vlki1w?file=/src/components/ImageWithState.tsx

When you create "imgElementArg" object in variable "rest" there is no "onError" prop ([here](https://github.com/vercel/next.js/blob/v12.1.5/packages/next/client/image.tsx#L790)) and, therefore [here](https://github.com/vercel/next.js/blob/v12.1.5/packages/next/client/image.tsx#L887) you have undefined

i found the problem, there is no onError in the "all" object ([here](https://github.com/vercel/next.js/blob/v12.1.5/packages/next/client/image.tsx#L371)).
When you create "imgElementArg" object in variable "rest" there is no "onError" prop ([here](https://github.com/vercel/next.js/blob/v12.1.5/packages/next/client/image.tsx#L790)) and, therefore [here](https://github.com/vercel/next.js/blob/v12.1.5/packages/next/client/image.tsx#L887) you have undefined

**Comment by balazsorban44** [comment]
You seem to be mixing Next.js and https://create-react-app.dev/

Your app is wired up to be a CRA app.

**Reply by danyabunkov** [reply]
I used "yarn create next-app --typescript" from official documentation

https://nextjs.org/docs/api-reference/create-next-app
